### DISCUSSION TITLE
Godot integration

### DISCUSSION THREAD

**Discussion by gamersalpha** [linkdd/aitoolkit#11] [Q&A]
Hello can i using it in godot please 

where found some how to to using it ?

**Comment by linkdd** [comment]
Hello there, since this is more of a Q/A kind of issue, I'm moving it to discussions 🙂

I'm not fluent with Godot, but if I recall correctly you can use C++ modules in Godot:

https://docs.godotengine.org/en/stable/contributing/development/core_and_modules/custom_modules_in_cpp.html

You would need to use this library to implement your Godot C++ module, and then use your module from GDScript etc...
I don't have much knowledge on how to do this, so sorry for only pointing you to the documentation of Godot.

I agree it would be nice to add a tutorial in the documentation for this integration (if someone wants to help?)
